caravel/config.py

```diff
@@ -113,7 +113,7 @@
 LANGUAGES = {
     'en': {'flag': 'us', 'name': 'English'},
     # 'fr': {'flag': 'fr', 'name': 'French'},
-    # 'zh': {'flag': 'cn', 'name': 'Chinese'},
+    'zh': {'flag': 'cn', 'name': 'Chinese'},
```

Comment:
yes. I try to translate which include mako template ,but also almost python files . for example .
`caravel/forms.py` 

```python
viz_type': SelectField(
                'Viz',
                default='table',
                choices=[(k, v.verbose_name) for k, v in viz_types.items()],
                description=_("The type of visualization to display")),
            'metrics': SelectMultipleSortableField(
                _('Metrics'), choices=datasource.metrics_combo,
                default=[default_metric],
                description=_("One or many metrics to display")),
            'metric': SelectField(
                _('Metric'), choices=datasource.metrics_combo,
                default=default_metric,
                description="Chose the metric"),
            'stacked_style': SelectField(
                _('Chart Style'), choices=self.choicify(
```

I fork the project and create `i18n` branch . I will send a new PR ASP